stable/cockroachdb/templates/cockroachdb-petset.yaml

```diff
@@ -80,13 +96,28 @@ spec:
         component: "{{.Release.Name}}-{{.Values.Component}}"
       annotations:
         helm.sh/created: {{.Release.Time.Seconds | quote }}
-        pod.alpha.kubernetes.io/initialized: "true"
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
Since it's effectively an unrecognized annotation in such clusters, it just gets ignored. It only affects scheduling if the cluster supports alpha features.